### DISCUSSION TITLE
Version 0.2.0 availability for Ubuntu 22.04

### DISCUSSION THREAD

**Discussion by poldolot** [X3n0m0rph59/eruption#186] [Q&A] [answered]
Hi @X3n0m0rph59 ! When will be version 0.2.0 available in PPA?

**Comment by X3n0m0rph59** [comment, marked as answer]
Hi @poldolot, I have published new packages for Eruption on all supported distros!

The Ubuntu PPA now has the latest packages `eruption` and `eruption-git` version `0.3.0`, going back up to (including) `Ubuntu Focal Fossa`.
